### DISCUSSION TITLE
Custom Table

### DISCUSSION THREAD

**Discussion by denison** [Laravel-Backpack/community-forum#589] [General] [answered]
I again, hahah

I need, when clicking on my module, to open a playlist with the videos in that way.

The video table already exists as you can see, I want to create a list for the new route just like backpack creates its beautiful lists. Has as?

https://github.com/Laravel-Backpack/community-forum/assets/6712969/f9e9bf8a-9ac0-4e7a-94d1-da38aa75b41d

Thanks a lot for the help

**Comment by christmex** [comment]
Hi @denison 

Can I know what is the relationship? I assume that you have hasMany relationships in your Module Class
So like 1 module has many videos, right?

**Reply by denison** [reply]
Yes, my Module hasMany relationship , 1 Module have many vídeos.

**Comment by christmex** [comment, marked as answer]
In my opinion, what if you just redirect to your video list, so from your module list, in your column name when you click make it redirect to video list

point it out so the **a href tag** will be something like this
```html
<a href="{{route('video.index')}}?module_id=1">The module name </a>
```

then inside your setupListOperation 
```php
    // this is VideoCrudController.php
    protected function setupListOperation()
    {
        // add these lines
        if(request('module_id')){
            CRUD::addClause('where','module_id','=',request('module_id'));
        }

        //Here's another code
    }
```

I hope you get the idea and I hope I'm not wrong to understand what you want to achieve mate.
Let me know if you found any issues.


Cheers.

**Reply by denison** [reply]
Hi @christmex  it's Perfect :). I didn't think that way. Thanks my friend.

**Reply by christmex** [reply]
Sure 🙌

**Comment by jcastroa87** [comment]
Hello @denison this look like you want to create a custom operation, in our [documentation](https://backpackforlaravel.com/docs/6.x/crud-operations#creating-a-new-operation-with-an-interface-1) there is a topic about how to create a custom operation with an interface and you can customize with blade as you need.

Cheers.

**Reply by denison** [reply]
@jorgetwgroup 

sorry if my question is stupid, I'm learning backpack. 

with getModerateForm method I create a form right? if I want to create a listOperation for example, which method to use? I didn't see anything about a route('moderate.index') for example....

But I understood the concept well, Thank you.

**Reply by jcastroa87** [reply]
Check on List documentation about [overwrite](https://backpackforlaravel.com/docs/6.x/crud-operation-list-entries#how-to-overwrite-1)
